docker/oracle/setup.sql

```diff
@@ -1,20 +1,23 @@
 alter profile default limit password_life_time unlimited;
 
+
+-- create tablespace BC_NUCORE
+--     DATAFILE 'bc_nucore.dat'
+--     SIZE 100M
+--     AUTOEXTEND ON;
+
```

Comment:
Do we need this comment? Can we get rid of it?

Reply:
It's needed for NU if you're loading a database dump, but we can probably get rid of it here.